src/main/java/cloud/orbit/spring/OrbitActorsProperties.java

```diff
@@ -28,19 +28,33 @@
 
 package cloud.orbit.spring;
 
-import cloud.orbit.actors.Stage;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
+import cloud.orbit.actors.Stage;
+import cloud.orbit.actors.extensions.ActorExtension;
+
 import java.util.List;
 
 @ConfigurationProperties(prefix = "orbit.actors")
 public class OrbitActorsProperties {
+    private String basePackages;
     private String clusterName;
     private String nodeName;
     private Stage.StageMode stageMode;
+    private List<ActorExtension> extensions;
```

Comment:
Another reason for this is that even though Spring supports YAML as a configuration source it expects the ability to represent the YAML or properties file as environment properties. It just happens that the current implementation of the YAML deserializer is using SnakeYAML and as such can support that instantiation.